doc/source/multivisit.rst

```diff
@@ -3,4 +3,535 @@
+
+All these definitions, including how finely each ProjectionCell should be divided to define the SkyCells, are encoded in a
+FITS table installed as part of the `drizzlepac` package:
+
+    drizzlepac/pars/allsky_cells.fits
+
+
+Defining each SkyCell
+----------------------
+Each ProjectionCell is split into a grid of 21 x 21 'sky cells' which serves as the most basic MVM product generated
+during MVM processing.  Sky cells, as defined as the SkyCell object in the code and referred to as **SkyCell**,
+are approximately 0.2° x 0.2° in size (~21500 x ~21500 pixels) and
+they have the same WCS as the ProjectionCell.  Each SkyCell gets identified by its position within the ProjectionCell
+as shown in this figure:
+
+.. figure:: images/SkyCell_numbering.png
+  :width: 90%
+  :alt: Numbering convention for SkyCells within a Projection Cell.
+
+  Numbering convention for SkyCells within a Projection Cell used for naming the SkyCell.
+
+The numbering of the SkyCells within a ProjectionCell starts in the lower left corner at (1,1) corresponding to the cell
+that with the lowest declination and largest RA since the SkyCell is oriented so that the Y axis follows a line of RA
+pointing towards the North Pole.
+This indexing provides a way to identify uniquely any position on the sky that can be used as the basis for a unique filename for
+all products generated from the exposures that overlap each SkyCell.  Mosaics generated for each SkyCell uses this
+indexing to create files with names using the convention:
+
+  skycell-p<PPPP>x<XX>y<YY>
+
+where:
+  ==========  =========================================================================
+  Element     Definition
+  ==========  =========================================================================
+  <PPPP>      ProjectionCell ID as a zero-padded 4 digit integer
+  <XX>,<YY>   SkyCell ID within ProjectionCell as zero-padded 2 digit integers
+  ==========  =========================================================================
+
+The WCS for each SkyCell gets defined as a subarray of the Projection cell's WCS.  This allows data across SkyCells in
+the same ProjectionCell to be combined into larger mosaics as part of the same tangent plane without performing any
+additional resampling.
+
+Code for Defining SkyCell ID
+'''''''''''''''''''''''''''''
+The code for interfacing with the cell definitions table can be imported in Python using:
+
+.. code:: python
+
+   from drizzlepac.haputils import cell_utils
+
+Determining what SkyCells overlap any given set of exposures on the sky can be done using the function:
+
+.. code:: python
+
+  sky_cells_dict = cell_utils.get_sky_cells(visit_input, input_path=None)
+
+where **visit_input** is the Python list of filenames of FLT/FLC exposures.  These files can be any set of FLT/FLC files
+and the WCS's defined in them will, by default, **be used as-is** to create the final combined mosaics.  The MVM products
```

Comment:
Change:
and the WCS's defined ==> and the WCS solutions defined

Reply:
Fixed in 3cd7f7e.